Dockerfile.no-systemd

```diff
@@ -0,0 +1,37 @@
+FROM balenalib/amd64-debian-node:10.16.3-buster-run
```

Comment:
@roman-mazur I think we should use the timestamp tag here like `balenalib/amd64-debian-node:10.16.3-buster-run-20190920` since `10.16.3-buster-run` is a moving tag and will be weekly rebuilt so likely to change or have an unexpected breaking change (can be from our side or upstream debian image). The timestamp tag will never be rebuilt so it's stable and we can just change to new timestamp if needed

Reply:
@Page- @mikesimos any thoughts on this?

Reply:
I would agree but mostly for reasons of being declarative, transparency and predictability, as once the base image is built and tagged, it could break if rebuilt later (it isn't expected to be the case, but still ). There is a simmilar change on a PR for the systemd base image, and a relevant conversation here. https://github.com/balena-io/open-balena-base/pull/94#discussion_r326556665

Reply:
I am massively on the fence about this, and see https://github.com/balena-io/open-balena-base/pull/94 for the reasonings. On the one hand pinning versions is declarative, on the other hand if we can't trust a rebuild of the base images, we have a far deeper problem that needs dealing with. My major point with pinning is 'Who then is assigned to frequently audit base releases so they don't get stale, instead of trying them and spotting the problems in balenaCI testing/staging?'.